input/intro-notes/StructureDefinition-implication-intro.xml

```diff
@@ -5,10 +5,20 @@
 	This portion of the implementation guide relies on the content in the <a href="general.html">General Genomic Reporting</a> and <a href="sequencing.html">Variant Reporting</a> portions of this implementation guide.  
   </p>
 
+<p>At the heart of many genomic reports are findings (<a href="StructureDefinition-variant.html">variants</a>, 
+<a href="StructureDefinition-haplotype.html">haplotypes</a>, 
+<a href="StructureDefinition-genotype.html">genotypes</a>, 
+<a href="StructureDefinition-molecular-biomarker.html">biomarkers</a>) and annotations derived from those findings (<a href="StructureDefinition-diagnostic-implication.html">diagnostic implications</a>, 
+<a href="StructureDefinition-therapeutic-implication.html">therapeutic implications</a>, 
+<a href="StructureDefinition-molecular-consequence.html">predicted molecular consequences</a>), as shown in the following figure:</p>
+<img src="Somatic_General_Guidance.svg" alt="generalGuidance" height="195" width="410"/>
+<p> </p>
```

Comment:
Actually, just realized this might be to ensure the text after the image doesn't wrap around the image. You could use a < br > for that, but this is OK as well. I will mark this as 'resolved'